apps/event-worker/src/workers/env-selector-update.ts

```diff
@@ -78,14 +80,19 @@ const createReleaseTargets = (
     )
     .onConflictDoNothing()
     .returning()
-    .then((releaseTargets) =>
-      getQueue(Channel.EvaluateReleaseTarget).addBulk(
+    .then(async (releaseTargets) => {
+      const createReleasePromises = releaseTargets.map((rt) =>
+        createRelease(db, rt, workspaceId),
```

Comment:
you shouldn't be creating releases in a function that is suppose to ` createReleaseTargets`